normalizer/tests/test_https_endpoint.py

```
@@ -0,0 +1,158 @@
+        test_address = "東京都中央区京橋1丁目19番13号"
+        
+        # 環境変数からベースURLを取得（未設定の場合はスキップ）
+        base_url = os.environ.get("JAGEOCODER_ENDPOINT")
+        
+        if not base_url:
+            print("[SKIP] JAGEOCODER_ENDPOINTが未設定のため、詳細テストをスキップします。")
+            return True  # テスト自体は成功とする
+        http_base = base_url.replace("https://", "http://")
+        https_base = base_url.replace("http://", "https://")
+        
+        for endpoint_name, endpoint_data in [
```

Ruff (0.12.2)

Loop control variable `endpoint_name` not used within loop body

Rename unused `endpoint_name` to `_endpoint_name`

(B007)